core/src/test/java/kafka/test/ClusterTestExtensionsTest.java

```diff
@@ -236,6 +236,20 @@ public void testNotSupportedNewGroupProtocols(ClusterInstance clusterInstance) {
         Assertions.assertEquals(1, clusterInstance.supportedGroupProtocols().size());
     }
 
+
+
+    @ClusterTest(types = {Type.ZK, Type.CO_KRAFT, Type.KRAFT}, brokers = 3)
+    public void testCreateTopic(ClusterInstance clusterInstance) throws Exception {
+        String topicName = "test";
+        int partitions = 3;
+        short replicas = 3;
+        clusterInstance.createTopic(topicName, partitions, replicas);
+
+        try (Admin admin = clusterInstance.createAdminClient()) {
+            Assertions.assertTrue(admin.listTopics().listings().get().stream().anyMatch(s -> s.name().equals(topicName)));
```

Comment:
Please check the partition and replica also

Reply:
Updated. Thanks for review.